packages/atomic/src/components/search/generated-answer/atomic-generated-answer/atomic-generated-answer.pcss

```diff
@@ -17,6 +32,16 @@
   background: var(--atomic-primary-background);
 }
 
+atomic-modal {
+  width: 100%;
+}
+
+.source-citations {
```

Comment:
Should also remove this. It's included in the imported `@import './styles/source-citations.pcss';`